fluent/syntax/parser.py

```diff
@@ -349,13 +349,14 @@ def get_number(self, ps):
     @with_span
     def get_pattern(self, ps):
         elements = []
-        ps.skip_ws()
+        ps.skip_inline_ws()
 
         while ps.current():
             ch = ps.current()
 
             if ch == '\n':
-                break
+                if not ps.is_peek_next_line_pattern():
+                    break
```

Comment:
@zbraniecki This fixed the parsing of the following FTL:

    key =
        { foo }
        Bar

Before this change, `get_pattern` would `break` after the closing `}`. Does this look right to you?

There's a bit of repetition between this code and `get_text_element`. I wonder if that's OK? An alternative would be to make `get_text_element` return `None` in some cases so that we can `break` in `get_pattern`. However that would be inconsistent with the return values of other `get_` methods, I think?